### DISCUSSION TITLE
Example to create custom datasets using rate and ISI spike encoding schemes

### DISCUSSION THREAD

**Discussion by adarshabalaji** [lava-nc/lava-dl#94] [Q&A]
Could you suggest a tutorial or example project that details the steps to convert my datasets into the spiking domain? 
I would like to use rate-based and ISI-based encoding with Slayer.

**Comment by bamsumit** [comment]
Hi @adarshabalaji , we don't have an ISI encoder built in SLAYER. So you will need to write your own, and contribute to the repo.

Depending on the type of your input, it could be just two spikes representing a numeric value or it could be a pulse frequency modulation.
